EventPlannerAPI/DataAccessLayer/Repositories/UserRepository.cs

```diff
@@ -4,6 +4,7 @@
 using Microsoft.AspNetCore.Identity;
 using DataAccessLayer.Helpers;
 using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
 
 namespace DataAccessLayer.Services
```

Comment:
Out of context
We have to change the namespace to be consistent
```suggestion
namespace DataAccessLayer.Repositories
```

Reply:
Are all DataAccessLayer.Services references updated?

Reply:
yes

Reply:
[AdminPage](https://github.com/tecknoworks/SportEventManager/pull/13/commits/9ebed5b4b139788bdffc0bc2eb8ce6be59612def)